### ISSUE TITLE
Question about the article Crash-avoiding Program Repair,what is the relationship between over-fit and crash

### ISSUE THREAD

**Issue by xyry** [gaoxiang9430/Fix2Fit#1]
the problem is as above,I didn't find the answer in the article,and i 'm confused about examples in your article's OVERVIEW. **Listing 1** and **Table 1**.
 " the fourth and sixth patches are semantically equivalent to the developers’ patch " I agree with this statement. BUT i am confused about the next sentence"However, other patches over-ft the existing test set." ,why they are over-fit
it is appreciated that if you answer me.

**Comment by zhiyufan** [comment]
It means such patches are not correct but just plausible patches (which pass all the tests in the given test suite). If there is a new test feed to the program,  it is still possible to fail it. For example, for the patch with id 1, if the input test made the remaining_space = width, it will fail, notice that such an input test is not in the give test suite. it is similar to the overfitting problem in machine learning, the given test suite is a training set and the new input test is test set

**Comment by xyry** [comment]
Thanks for your reply!
